### ISSUE TITLE
Remove meta tag "maximum-scale=1.0" disabling scaling

### ISSUE THREAD

**Issue by benlk** [INN/Largo#1785]
This issue was identified during research on https://github.com/INN/largo/issues/1653 using the AXE browser plugin.

https://github.com/INN/largo/blob/512da701664b329f2f92244bbe54880a6e146431/header.php#L10

> To solve this violation, you need to:
> Fix the following:
> 
> - maximum-scale on <meta> tag disables zooming on mobile devices

- [ ] remove that value
- [ ] test that zooming is enabled on mobile devices that do not have the setting enabled to allow zooming despite website preferences (like Ben's phone)

**Comment by MirandaEcho** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Remove meta tag "maximum-scale=1.0" disabling scaling](https://trello.com/c/O3BXEt3m/14-remove-meta-tag-maximum-scale10-disabling-scaling)
